### ISSUE TITLE
Revive x32 packages?

### ISSUE THREAD

**Issue by svenha** [fantix/ArchLinux-x32#5]
It is a pity that this excellent set of x32 packages did not inspire more people to use the x32 ABI.

Is there a chance to revive a minimal subset of x32 packages so that one compile a minimal C program? (i.e. libx32-glibc and surrounding packages)

Is this right repo to discuss this?

**Comment by fantix** [comment]
Feel free to do so! I'd be happy to grant access to the https://github.com/archlinux-x32 organization because I'm likely unavailable to review PRs for those repos.

**Comment by svenha** [comment]
Unfortunately, I have not enough Arch expertise, yet, to pick this up. I will try to learn more about Arch and gather some other interested people.

If somebody is interested and/or can help, please leave a note here.

**Comment by svenha** [comment]
If I understand correctly, a minimal installation would need these repos (in updated versions):

- gcc-x32-seed
- glibc-x32-seed 
- libx32-glib
- gcc-multiilib-x32

Anything else?
Is this order correct?

**Comment by fantix** [comment]
Yeah, you may need glibc-x32-seed (just a stub) before the gcc-x32-seed. Then libx32-glibc and gcc-multiilib-x32 will replace the corresponding seeds.
